src/reaper_osara.cpp

```diff
@@ -1176,11 +1176,17 @@ void clickIoButton(MediaTrack* track, bool rightClick=false) {
 
 bool maybeSwitchToFxPluginWindow() {
 	HWND window = GetForegroundWindow();
-	char name[4];
+	char name[8];
 	if (GetWindowText(window, name, sizeof(name)) == 0)
 		return false;
-	if (strncmp(name, "FX: ", 4) != 0)
+	if (strncmp(name, "FX: ", 4) != 0 &&
+		strncmp(name, "DX: ", 4) != 0 &&
```

Comment:
Can you please add a comment above this line noting that the following checks are for floating FX windows?